Automattic/jetpack
# Remove wp-admin warning for Writing/Discussion settings

Fixes https://github.com/Automattic/dotcom-forge/issues/5319, https://github.com/Automattic/dotcom-forge/issues/5322

## Proposed changes:
After a settings audit, all settings are still available in"Classic" mode (`'wp-admin' === get_option( 'wpcom_admin_interface' )`). We can safely remove the warning banner.

![image](https://github.com/Automattic/jetpack/assets/6586048/f98ad060-7eda-4a05-8a66-24536e466010)


### Other information:


## Jetpack product discussion


pfsHM7-J-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Use the Jetpack Beta plugin to load this branch to your trunk
* Make sure the banner doesn't show for `/wp-admin/options-discussion.php` and `/wp-admin/options-writing.php` when atomic site is in "Classic" Mode, you can set it in `/hosting-config/:site`
